UI/src/app/pages/terms/terms.component.html

```diff
@@ -0,0 +1,109 @@
+<link rel="stylesheet" href="../about/about.component.scss">
+<script src="terms.component.ts"></script>
```

Comment:
the same here, you don't need to load it, angular knows which .ts file is combined to this html.